### DISCUSSION TITLE
Error: 8: Failed to execute 'insertBefore' on 'Node': The node before which the new node is to be inserted is not a child of this node.

### DISCUSSION THREAD

**Discussion by Ecmpython** [AntlerVC/firetable#1019] [Support / Q&A] [answered]
👉 **Please describe the steps that you took that led to this bug.** i have tried to create a project and it always shows me in that comment

**Comment by BeeBombshell** [comment, marked as answer]
Hey @Ecmpython, does the issue still persist?

**Reply by BeeBombshell** [reply]
Hey @Ecmpython, closing this discussion since it has been abandoned for a long time. Hope this issue has now been resolved for you. If not, feel free to create a new one.
